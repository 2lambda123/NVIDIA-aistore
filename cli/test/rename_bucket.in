ais gen-shards --bucket $BUCKET_1 --ext .tar --template="tmp/shard-{0..9}" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais rename bucket $BUCKET_1/ $BUCKET_2/
ais rm bucket $BUCKET_2

ais rename bucket cloud://bucket new_bucket // FAIL "Renaming cloud buckets (cloud://bucket) is not supported"
ais rename bucket bucket cloud://new_bucket // FAIL "Renaming cloud buckets (cloud://new_bucket) is not supported"
ais rename bucket aws://bucket new_bucket // FAIL "Renaming cloud buckets (aws://bucket) is not supported"
