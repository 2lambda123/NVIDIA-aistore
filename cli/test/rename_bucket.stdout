Bucket "$BUCKET_2" doesnotexist
To check the status, run: ais show xaction renamelb $BUCKET_2
$BUCKET_2 bucket destroyed
